docs/source/unit-testing.md

```diff
@@ -200,3 +202,33 @@ Note: Dependencies defined the way mentioned above is for test execution only.
 If your actor requires any package when executed as part of a workflow, it
 needs to be specified in a
 [leapp-repository specfile](https://github.com/oamg/leapp-repository/blob/master/packaging/leapp-repository.spec).
+
+### Shared libraries' tests
+
+To run tests of shared libraries (i.e. libraries stored in
+`repositories/system_upgrade/el7toel8/libraries`) environment variable
+`LEAPP_TESTED_LIBRARY` with path of the shared library needs to be specified for
+_pytest_ to be able to import leapp modules.
+
+
+To run tests for all shared libraries:
+```sh
+LEAPP_TESTED_LIBRARY="$PWD/libraries" pytest -sv libraries/tests/test_tcpwrappersutils.py
```

Comment:
what is the difference between this call and the one bellow? shouldn't the path be `libraries/tests` only?

Reply:
Good catch, fixed.